cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,46 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import CartPageObject from '../support/pages/cart.pageObject';
+const homeAndCataloguePageObject = new HomeAndCataloguePageObject();
+const cartPageObject = new CartPageObject();
 
-  });
+describe('', () => {
+  before(() => {});
```

Comment:
unnecessary code, delete it or use here the "visit" command

Reply:
I completely forgot about it. thanks for reminding me.